packages/cli/__tests__/zosfiles/__unit__/compare/CompareBaseHelper.unit.test.ts

```diff
@@ -63,15 +63,15 @@ describe("Comapare Base Helper", () => {
             jest.restoreAllMocks();
         });
 
-        it("should get the contents of a local file", () => {
+        it("should get the buffer of a local file", () => {
             jest.spyOn(fs, "openSync").mockReturnValue(0);
             jest.spyOn(fs, "fstatSync").mockReturnValue({isFile: () => true} as any);
             jest.spyOn(fs, "readFileSync").mockReturnValue("test");
             jest.spyOn(fs, "closeSync").mockImplementation();
 
             const response = helper.prepareLocalFile("/absolute/path/to/real/file");
 
-            expect(response).toEqual("test");
+            expect(response).toEqual(Buffer.from("test"));
         });
```

Comment:
I think the changes to this test can be reverted to use a string rather than a buffer, since you restored the implementation of the `CompareBaseHelper` method to the previous implementation 🙂